shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -257,6 +257,24 @@ public int getHoveredObjectId() {
 
   @Nullable private OnAccessibilityChangeListener onAccessibilityChangeListener;
 
+  // Whether there is assitive technology[s] running.
+  //
+  // Use the setter to update this property if needed.
+  @VisibleForTesting public boolean hasAssistiveTechnology = false;
+
+  private void setHasAssistiveTechnology(boolean value) {
```

Comment:
nitpick: Maybe call this `updateHasAssistiveTechnology`. Alternatively, require `sendLatestAccessibilityFlagsToFlutter` to take a boolean and have it set the flags. Then in tests you don't need the extra flag taken.

Reply:
I meant to make this a setter of hasAssistiveTechnology. I am very lazy so I just search online, this is the common way people name it. WDYT?

 sendLatestAccessibilityFlagsToFlutter sends the entire feature flags over, unless I create an API specific for `ACCESSIBLE_NAVIGATION` it feels weird to add an parameter to sendLatestAccessibilityFlagsToFlutter for `ACCESSIBLE_NAVIGATION` only.